aead/src/lib.rs

```diff
@@ -140,13 +142,15 @@ pub trait AeadCore {
     ///
     /// [NIST SP 800-38D]: https://csrc.nist.gov/publications/detail/sp/800-38d/final
     #[cfg(feature = "rand_core")]
-    fn generate_nonce(mut rng: impl CryptoRng + RngCore) -> Nonce<Self>
+    fn generate_nonce_with_rng(
```

Comment:
Maybe we also should add `generate_nonce` method similarly to `crypto-common`?

Reply:
Added in [963a8cf](https://github.com/RustCrypto/traits/pull/1384/commits/963a8cf5aee061effb5541c2d95c5b3c25e98f96)